support/cas-server-support-x509-webflow/src/main/java/org/apereo/cas/web/flow/X509WebflowConfigurer.java

```diff
@@ -31,6 +31,11 @@ protected void doInitialize() throws Exception {
                     CasWebflowConstants.TRANSITION_ID_WARN));
             actionState.getTransitionSet().add(createTransition(CasWebflowConstants.TRANSITION_ID_ERROR,
                     CasWebflowConstants.STATE_ID_VIEW_LOGIN_FORM));
+            // auth failure handling case where CAS got a valid certificate but couldn't find entry in attribute
```

Comment:
This is great. My only comment is: avoid inlined comments. Turn these into javadocs, so they can reviewed externally, formatted and linked to APIs, etc.